utilities/transactions/transaction_lock_mgr.h

```diff
@@ -27,7 +27,7 @@ struct LockMap;
 struct LockMapStripe;
 
 class Slice;
-class TransactionDBImpl;
+class PessimisticTxnDB;
```

Comment:
Transaction makes the name child classes too long. I can change PessimisticTransaction to PessimisticTxn though.